src/main/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommand.java

```diff
@@ -1,17 +1,100 @@
 package seedu.sellsavvy.logic.commands.ordercommands;
 
+import static java.util.Objects.requireNonNull;
+import static seedu.sellsavvy.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.sellsavvy.logic.parser.CliSyntax.PREFIX_COUNT;
+import static seedu.sellsavvy.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.sellsavvy.logic.parser.CliSyntax.PREFIX_ITEM;
+
+import java.util.List;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.commons.util.ToStringBuilder;
+import seedu.sellsavvy.logic.Messages;
 import seedu.sellsavvy.logic.commands.Command;
 import seedu.sellsavvy.logic.commands.CommandResult;
 import seedu.sellsavvy.logic.commands.exceptions.CommandException;
 import seedu.sellsavvy.model.Model;
+import seedu.sellsavvy.model.order.Order;
+import seedu.sellsavvy.model.order.OrderList;
+import seedu.sellsavvy.model.person.Person;
 
 /**
  * Adds an order under a specified person.
  */
 public class AddOrderCommand extends Command {
 
+    public static final String COMMAND_WORD = "addOrder";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an order under the specified person. "
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_ITEM + "ITEM "
+            + PREFIX_DATE + "DELIVERY_BY "
+            + PREFIX_COUNT + "ITEM_COUNT\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_ITEM + "Lamp "
+            + PREFIX_DATE + "20-12-2024 "
+            + PREFIX_COUNT + "2";
```

Comment:
Default count might need to be handled for omitted count values when adding orders?

Reply:
will handle in separate issue